### PR TITLE
Add filter status field in table

### PR DESCRIPTION
**Pull request by whDongRui**
- add delete method in detail page
- add id copy sucess tips

### CODE REVIEW

File: src/pages/Admin/Apps/Detail/index.jsx [hunk 1]
```diff
@@ -35,8 +36,18 @@ export default class AppDetail extends Component {
     this.loginUser = getSessInfo('user', props.sessInfo);
   }
 
+  componentDidUpdate() {
+    const { appDetail, deleteResult } = this.props.appStore;
+    console.log(appDetail.status, deleteResult);
```
Comment (iwisunny) [inline src/pages/Admin/Apps/Detail/index.jsx]: remove console.log

File: src/pages/Admin/Runtimes/Detail/index.jsx [hunk 2]
```diff
@@ -105,6 +107,22 @@ export default class RuntimeDetail extends Component {
         render: item => <TimeShow time={item.status_time} />
       }
     ];
+    const filterList = [
```
Comment (iwisunny) [inline src/pages/Admin/Runtimes/Detail/index.jsx]: this block is deplicate, can be moved to one module

File: src/pages/Admin/Apps/Detail/index.jsx (hunk 1 above)
Comment (whDongRui) [reply src/pages/Admin/Apps/Detail/index.jsx]: ok

File: src/pages/Admin/Runtimes/Detail/index.jsx (hunk 2 above)
Comment (whDongRui) [reply src/pages/Admin/Runtimes/Detail/index.jsx]: the file has only one definition about filterList.

**Pull request merged**